_data/toc.yaml

```diff
@@ -1104,10 +1104,10 @@ manuals:
       title: Binaries
     - path: /install/linux/linux-postinstall/
       title: Optional Linux post-installation steps
-  - sectiontitle: Docker Engine
+  - path: /engine/release-notes/
+    title: Release notes
+  - sectiontitle: Prior releases
```

Comment:
How about `Previous versions` instead of Prior releases?

Reply:
Heh: I actually had that initially, but changed to "prior" as I saw that's what a tech-writer changed it to in the past.

I'll update